yarn-project/aztec.js/src/contract/contract_function_interaction.ts

```diff
@@ -23,6 +23,8 @@ export type SimulateMethodOptions = {
   from?: AztecAddress;
   /** Gas settings for the simulation. */
   gasSettings?: GasSettings;
+  /** Option to throw an error if simulation does not produce a valid transaction. */
+  ignoreInvalidTx?: boolean;
```

Comment:
Hm better, but this makes me thing that an invalid tx would be ignored - which is actually the opposite of what this does, we 'ignore' the fact that it is invalid.

What about 'skipTxValidation'?